Ignore event lines in event stream.

xperimental opened the pull request:
The event stream seems to have three types of "lines":
- Empty lines sent in regular interval as "heartbeat"
- Lines starting with "event: " containing only the event type
- Lines starting with "data: " containing a JSON representation of the event

Currently the "event: " lines produce log output hinting that there was an error while they can be safely ignored (the event type is also present in the JSON).

Fixes #219

swagatha-christie commented:
By analyzing the blame information on this pull request, we identified @activars, @bsideup and @g0hacker to be potential reviewers

bsideup commented:
Since Marathon is keep adding the new event types, maybe, instead of filtering, we should change it to: "if starts with "data" then process" ?

xperimental commented:
@bsideup Added clarification to the PR description.

bsideup commented:
@xperimental with your clarification it makes even more sense then :)

xperimental commented:
Updated after merging #226. I removed the `t.Parallel()` again because I had tests which should fail be successful, it seems like my guess that it should be ok with parallelizing httptest was wrong.

xperimental commented:
Updated again, this time with a fix for the parallel tests. Turns out it was not an issue with httptest but rather with me not reading the documentation properly.

bluepeppers commented:
Thanks :)

activars commented:
🌮

xperimental commented:
You're welcome. Thanks for merging.
